.github/PULL_REQUEST_TEMPLATE/feature_template.md

```diff
@@ -0,0 +1,40 @@
+# New Feature 
+**Name:**
+
+**module:** *python file name*
+
+**package:** *folder it is under*
+
+## Issue reference
+*Does this feature pertain to a specific issue?*
+
+**Issue** *number sign and number of the issue*
+
+## Scope
+*What does this feature do?*
+
+*What does this feature not do?* 
+
+### Requirements Checklist
+- [ ] PEP 8/flake8 check
+- [ ] unit tests
+- [ ] Code documentation
+- [ ] readthedocs documentation
+- [ ] merge to `develop`
+
+## Test code
+*Please provide an example of code to show how to test it*
+
+*Please provide **expected output** of the code provided for testing*
+
+### Where to obtain test files
+*Please provide instructions if applicable on how to access files needed to test the code*
+
+## Extra notes
+*Please provide any other information on this PR here.*
+
+*Example info:*
+- *does it use metadata?*
+- *does it require a change in the `setup.py`*
+- *are there performance hits/improvements?*
+- *what is the design pattern?*
```

Comment:
Also recommend adding the python version and operating system used for development here.